### ISSUE TITLE
Set screen placement per application

### ISSUE THREAD

**Issue by SpyroTEQ** [fullphat/snarl#10]
Currently, you can set which screen notifications should be shown on, for the whole SNARL (Settings/Notification/Layout/screen 1, 2, 3, 4, etc).
Currently, you can set the on-screen position for the whole snarl, and override it per application (top/bottom & left/right).
Currently, you cannot set on which screen an application should display its notifications.

So, I cannot set an application to display its notifications on the bottom-right corner of screen 2, while all other applications notifications are displayed on the bottom-right corner of screen 1 (Primary).

(btw, I just love Snarl and its API)

**Comment by sleprevost** [comment]
Will those features be available in R5 ?
At this time, looks like ALL notifications are shown on screen number 1, which can be (or not) the main screen.
A colleague of mine has screen 2 as main screen, which is the one he uses when connecting using remoste desktop. That means he can't see notifications when connecting remotely :-(

**Comment by fullphat** [comment]
Yes, and now Beta 2 is out, focus has shifted to OSD improvements.  Already it's now possible to change the colour, font and some other features of the notifications.  Screen placement will also be reinstated.

I am debating the use of multiple styles as I've always found this a bit cumbersome, but I've seen other requests on here that show this feature is still used (@blackwind).
